draft-ppm-protocol.md

```diff
@@ -1142,6 +1145,17 @@ invalid input.
    shares.
 1. If computed over a sufficient number of input shares, output shares reveal
    nothing about either the inputs or the participating clients.
+1. Clients can ensure that aggregate counts are non-sensitive by generating
+   input independently of user behavior.
+1. Bogus inputs can be generated that encode "null" shares that do not affect
+   the aggregate output, but mask the total number of true inputs.
+     * Either leaders or clients can generate these inputs to mask the total
+       number from non-leader aggregators or all the aggregators, respectively.
+     * In either case, care must be taken to ensure that bogus inputs are
+       indistinguishable from true inputs (size, metadata, etc).
```

Comment:
```suggestion
       indistinguishable from true inputs (metadata, etc), especially when
       constructing timestamps on reports.
```
The size will necessarily be the same as a true input as we discussed earlier.

Reply:
Yes that's right